web_ui/ui.go

```diff
@@ -576,6 +576,20 @@ func configureCommonEndpoints(engine *gin.Engine) error {
 	engine.DELETE("/api/v1.0/deleteApiToken/:id", deleteApiToken)
 	engine.GET("/api/v1.0/listApiTokens", listApiTokens)
 	engine.GET("/api/v1.0/version", getVersionHandler)
+
+	isOriginOrCache := config.IsServerEnabled(server_structs.OriginType) || config.IsServerEnabled(server_structs.CacheType)
+	if isOriginOrCache {
+		downtimeAPI := engine.Group("/api/v1.0/downtime")
+		{
+			downtimeAPI.POST("/", AuthHandler, AdminAuthHandler, HandleCreateDowntime)
+			downtimeAPI.GET("/", AuthHandler, AdminAuthHandler, HandleGetIncompleteDowntime)
+			downtimeAPI.GET("/all", AuthHandler, AdminAuthHandler, HandleGetAllDowntime)
```

Comment:
```suggestion
			downtimeAPI.GET("/all", AuthHandler, AdminAuthHandler, HandleGetAllDowntime)
```

I would prefer this didn't exist. REST best practices indicates that filtering should be done via url parameters and not entirely new endpoints. 

I would prefer something more along the lines of a single GET endpoint that accepts `active=true` or active=false` to filter the response.